CONTRIBUTING.md

```diff
@@ -0,0 +1,121 @@
+# Contribution guide
+
+![github-profile](https://user-images.githubusercontent.com/10350960/166113119-629295f6-c282-42c9-9379-af2de5ad4338.png)
+
+- [Contribution guide](#contribution-guide)
+  - [Ways to contribute](#ways-to-contribute)
+  - [Finding an issue](#finding-an-issue)
+  - [Ask for help](#ask-for-help)
+  - [Pull request lifecycle](#pull-request-lifecycle)
+  - [Setting up the development environment](#setting-up-the-development-environment)
+    - [Forking and cloning the project](#forking-and-cloning-the-project)
+      - [Prerequisites](#prerequisites)
+  - [Signing commits](#signing-commits)
+
+Welcome 👋 Thank you for your interest in contributing to our project. We are happy to have you join us! 💖
+
+As you get started, you are in the best position to give us feedback on project areas we might have forgotten about or assumed to work well. These include, but are not limited to, the following:
+
+- Problems found while setting up a new developer environment
+- Gaps in our documentation
+- Bugs in our automation scripts
+
+If anything doesn't make sense or doesn't work as expected, please open an issue and let us know!
```

Comment:
Clarify between when to open an issue and when the start with a discussion first.